views/js/controller/pciManager/index.js

```diff
@@ -0,0 +1,28 @@
+define([
+    'util/url',
+    'qtiItemPci/pciManager/pciManager',
+    'css!qtiItemPciCss/pci-manager'
+
+], function (url, pciManager) {
+    'use strict';
+
+    var $container = $('.pci-manager');
+
+    var indexController = {
```

Comment:
No need to store `indexController` as a var here. You can simply return the object from this line. A matter of preference, really.